Doc/whatsnew/3.11.rst

```diff
@@ -70,6 +70,75 @@ Summary -- Release highlights
 New Features
 ============
 
+.. _whatsnew311-pep657:
+
+Enhanced error locations in tracebacks
+--------------------------------------
```

Comment:
We need to mention here at the end of this section how to deactivate it. Basically a very similar note as:

```
         .. note::
            Storing positions may increase the disk usage of compiled Python
            files and memory usage of the interpreter. This feature can be
            disabled with the ``-X no_debug_ranges`` command line flag.
```

Reply:
Added as a `.. note::`. Can also change to a paragraph if that is better.